mkt/websites/management/commands/import_games_from_csv.py

```diff
@@ -125,10 +125,11 @@ def create_instances(self, data):
                         self.set_promo_imgs(website, row)
                     except Exception as e:
                         print e
+                        WebsiteIndexer.refresh_index()
                         website.delete()
                         raise e
 
-                    WebsiteIndexer.index([website.id], no_delay=True)
+                    WebsiteIndexer.index_ids([website.id], no_delay=True)
```

Comment:
If you move that after the `self.set_promo_imgs(website, row)` you don't need 2 index calls. Also, as above, `index_ids` does not take a `no_delay` argument